orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -59,6 +68,40 @@ <h3>
                 </button>
               </td>
             </tr>
+            <tr ng-if="teamInfoCard.isProjectAdmin">
+              <td colspan="5">
+                <a
+                  class="btn"
+                  ng-click="teamInfoCard.toggleShowUnassigned()">
+                  {{teamInfoCard.showUnassigned ? 'Hide unassigned' : 'Show unassigned' }}
+                </a>
+              </td>
+            </tr>
+            <tr
+              ng-if="teamInfoCard.showUnassigned"
+              ng-repeat="unassigned in teamInfoCard.unassigned">
+              <td>{{unassigned.role}}</td>
+              <td>
+                <input type="text"
+                  placeholder="Username"
+                  ng-keydown="$event.keyCode === 13 && teamInfoCard.handleKeydown(unassigned.task_id, '', unassigned.stepSlug, unassigned.worker)"
+                  ng-model="teamInfoCard.assignmentInput[unassigned.stepSlug]" />
+                <button
+									class="btn btn-default btn-xs"
+                  ng-disabled="teamInfoCard.isBtnDisabled(unassigned.stepSlug)"
+                  ng-if="teamInfoCard.isProjectAdmin"
```

Comment:
do you need this since it's hidden by the `ng-if` in the `tr` above?

should you check if this task is staffable?